### ISSUE TITLE
save Workspace is a json filled with nulls

### ISSUE THREAD

**Issue by laurentopia** [iNavFlight/blackbox-log-viewer#32]
Is this function implemented? If so, what is its purpose?
Before I close a BB log session I save the workspace, hoping that it saves log to video offset but the file generates is a null json

> {
    "graphConfig": [
        null,
        null,
        null,
        null,
        null,
        null,
        null,
        null,
        null,
        null
    ]
}

**Comment by Tonynux** [comment]
And after 5 years we are at the same point!

**Comment by DzikuVx** [comment]
yes @Tonynux you are more than welcome to work on this problem just like any other open source user.

**Comment by Tonynux** [comment]
@DzikuVx yes, I understand your point of you, and of course you are right! I would like to be able to solve this problem, but unfortunately I am not.
Pawel, may you please just paste here some workspace.json file template, so anyone can manually customize the file to be imported il Log-Viewer?!

**Comment by druckgott** [comment]
Fixed it:
https://github.com/iNavFlight/blackbox-log-viewer/pull/99
